src/tango/quic/fd_quic.h

```diff
@@ -478,7 +476,8 @@ fd_quic_footprint( fd_quic_limits_t const * limits );
 
 FD_QUIC_API void *
```

Comment:
Can you add a comment here about the ip ownership changing. Something like: `fd_quic_new` retains a read/write interest in `ip` that persists until the memory is unformatted.